pctiler/setup.py

```diff
@@ -8,20 +8,19 @@
     "jinja2==3.0.3",
     "pystac==1.*",
     "planetary-computer==0.4.*",
-
     "rasterio==1.3.*",
     "titiler.core==0.10.2",
     "titiler.mosaic==0.10.2",
-
     # titiler-pgstac
     "psycopg[binary,pool]",
     "titiler.pgstac==0.2.2",
-
     # colormap dependencies
     "matplotlib==3.4.*",
-
     "importlib_resources>=1.1.0;python_version<'3.9'",
     "pccommon",
+    "xarray",
+    "rioxarray",
+    "zarr",
```

Comment:
```suggestion
    "zarr",
    "fsspec",
```

apparently you need `zarr` to read `zarr` but you also need `fsspec` :-)